releasenotes/notes/vqc-input-validation-d5550e6bcd531f22.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    :class:`~qiskit_machine_learning.algorithms.classifiers.vqc.VQC` will now
```

Comment:
Can you check in the built documentation that this resolves as a link (and the one below), i.e. you can click it and it takes you to the class. Since its building still in the CI I cannot at present. If you have not done this go to any build Details so you are in the CI results and goto Summary. Then scroll down and you should see a list of build artifacts (zip files) One is the `documentation` so if you download and unzip/untar that you will end up with the built docs. You can start out in the index.html if you like or go directly to the release_notes.html and find this entry there. I am thinking it will need to be
``:class:`~qiskit_machine_learning.algorithms.VQC` `` as this is where we document this class as its imported at this level for users convenience.